WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -229,7 +229,7 @@ protected SendControlViewModel(Wallet wallet, string title)
 					LabelSuggestion.Label = label;
 					if (!IsMax && label.IsEmpty)
 					{
-						NotificationHelpers.Warning("Observers are required.", "");
+						NotificationHelpers.Warning("Observers are required.", "", sender: Wallet);
```

Comment:
Same here, it is unnecessary. However, it could make sense if we decide to add the wallet name to absolutely all alerts.

Reply:
Removed all